README.md

```diff
@@ -355,11 +343,10 @@ OpenTracing standardized tags are mapped to AppOptics tag names as well.
 
 To set AppOptics's tracer to be your global tracer, call `opentracing.InitGlobalTracer(opentracing.NewTracer())`.
 Currently, `opentracing.NewTracer()` does not accept any options, but this may change in the future.
-Please let us know if you are using this package while it is in preview by contacting us at opentracing@tracelytics.com.
+Please let us know if you are using this package while it is in preview by contacting us at support@appoptics.com.
 
 ## License
 
-Copyright (c) 2017 Librato, Inc.
+Copyright (c) 2018 Librato, Inc.
 
 Released under the [Librato Open License](http://docs.traceview.solarwinds.com/Instrumentation/librato-open-license.html), Version 1.0
```

Comment:
@cce I don't think it's ideal to link to TraceView docs here.  Do we have a better hosted version/license for this yet?  If not, propose we change link in readme to `LICENSE` file in this repo?

Reply:
cc @leecalcote

Reply:
I checked when I was scrubbing all mentions of TraceView but noticed that the LOL license hasn't been copied into the AO KB repo. If it were, we could update the link.